src/runtime/ram-disk-memory-provider.ts

```diff
@@ -0,0 +1,15 @@
+/**
+ * @license
+ * Copyright (c) 2019 Google Inc. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * Code distributed by Google as part of this project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {VolatileMemory} from './storageNG/drivers/volatile.js';
+
+export interface RamDiskMemoryProvider {
```

Comment:
Is RamDiskMemoryProvider a best name? WDYT about VolatileMemoryProvider? Seems more high level.

Reply:
Renamed to VolatileMemoryProvider and moved into `storageNG/drivers/volatile.ts`.